actions-server/package.json

```diff
@@ -4,19 +4,34 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "start": "nodemon -r esm src/server.js"
+    "start": "yarn run serve",
+    "build": "yarn run build-ts",
+    "serve": "node build/actions-server/src/server.js",
+    "build-ts": "tsc",
+    "watch-ts": "tsc -w",
+    "debug": "npm run build && npm run watch-debug",
+    "serve-debug": "nodemon --inspect build/actions-server/src/server.js",
+    "watch-debug": "concurrently -k -p \"[{name}]\" -n \"TypeScript,Node\" -c \"cyan.bold,green.bold\" \"npm run watch-ts\" \"npm run serve-debug\""
```

Comment:
largely copied from https://github.com/microsoft/TypeScript-Node-Starter/blob/master/package.json

Reply:
way simplified in 3eb2636